course_content/notebooks/iris_intro.ipynb

```diff
@@ -567,25 +523,21 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": []
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "3\\. Extract the `sea_water_potential_temperature` cube. Does this cube have lazy data? Print the minimum, maximum, mean and standard deviation of the cube's data. Does the cube still have lazy data?"
+    "3\\. Index `cubes` to retrieve the `sea_water_potential_temperature` cube. Does this cube have lazy data? Print the minimum, maximum, mean and standard deviation of the cube's data. Does the cube still have lazy data?"
```

Comment:
Hmmm. I'll take a look at this during the course - I might revert this one as we don't want to encourage this form when ``cube.extract('sea_water_potential_temperature')`` is so much more maintainable.

Reply:
@pelson indeed it is, but at this point in the course we haven't introduced extraction. What we repeatedly observed was that course delegates looked ahead in the notebook and tried to apply `cubelist.extract` to this situation, which looks bad as it looks like the notebook is poorly ordered (when it's not), and is bad because inevitably it causes confusion as they're trying to apply a concept that they (deliberately) haven't been taught.

Instead, by replacing the word "extract" with "index", you:
* make it clear how the delegate is expected to try to solve part of the exercise,
* demonstrate that cubelists can be indexed, whether or not it is a good idea, and
* get the chance to make an extra teaching point when we _actually_ introduce `extract`, through showing the benefit of extract over cubelist indexing by a sort of compare-and-contrast.